pithos/pithos.py

```diff
@@ -22,13 +22,24 @@
 import signal
 
 import gi
-gi.require_version('Gst', '1.0')
 from gi.repository import Gst, GstPbutils, GObject, Gtk, Gdk, Pango, GdkPixbuf, Gio, GLib
 import contextlib
 import html
 import math
 import urllib.request, urllib.error, urllib.parse
 import json
+
+try:
+    gi.require_version('Gst', '1.0')
+except ValueError, e:
+    if e[0].startswith("Namespace Gst not available for version"):
+        print "Received error %s from Gstreamer." % e[0]
```

Comment:
Actually I didn't realize at all that this was related. The default Python on my system that I used for python setup.py build etc is python 2.7. Your build seems to have zero checks for versions or missing packages, so I do consider it broken, at least to some extent, certainly not very inviting to potential contributors. :)